src/Console/Application.php

```diff
@@ -51,6 +52,8 @@ public function __construct()
         $this->filesystem = new Filesystem();
         $this->container = $this->getContainer();
         $this->setDispatcher($this->container->get('event_dispatcher'));
+        $this->container->set('console.input', new ArgvInput());
```

Comment:
This won't work since the output / input wont be configured meaning the verbosity level is not available on the output in the service `grumphp.io.console` (which is still marked as lazy). The `grumphp.io` is required before the `configureIO()` method was called.
You can validate this by running the code with the verbose flag `-v`: You won't see the additional information.